sw/CacheChildrenStrategy.ts

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable import/no-unused-modules */
+import { Strategy } from 'workbox-strategies';
+
+/**
+ * This is a custom strategy to cache the milestone children of a Starmap.
+ * The benefit of writing this as a workbox strategy is we can use other workbox plugins like expiration.
+ */
+export class CacheChildren extends Strategy implements Strategy {
+
+  /**
+   *
+   * @param {string} cacheKey
+   * @param {object} request
+   * @param {object} handler
+   * @returns
+   */
+  async populateCacheAsync(cacheKey, request, handler) {
+    const response = await handler.fetch(request)
+    handler.cachePut(cacheKey, response.clone())
+    return response
+  }
+
+  /**
+   *
+   * @param {object} request
+   * @param {object} handler
+   * @returns
+   */
+  async _handle(request, handler) {
+    try {
+
+      // Cloning ensures we don't consume the request here.
+      const { issue_number, owner } = await request.clone().json()
+      // We are using the owner and issue number as the cache key.
+      const cacheKey = `child-${owner}-${issue_number}`
```

Comment:
owner for orgs would be `ipfs`, so repos with the same issue number may return improper cache. we should cache on github-short-id. e.g. `${owner}/${repo}#${issue_number}`